packages/frontend/src/pages/Tile/index.tsx

```diff
@@ -53,8 +47,16 @@ export default function Tile(): JSX.Element {
       gap={4}
       p={8}
     >
-      <Text textStyle="h4">{name}</Text>
-      <Table tableId={id} tableColumns={columns} tableRows={rows} />
+      <Text textStyle="h4" cursor="pointer">
+        {name}
+      </Text>
+      <TableContextProvider
+        tableId={id}
+        tableColumns={columns}
+        tableRows={rows}
+      >
+        <Table />
```

Comment:
nit: since `Table` immediately uses this context, any change in `id` (unlikely), `columns` (semi-likely) or `rows` (almost guaranteed) will cause a full re-render of `Table`.

We discussed this perf problem today and we landed on memoing as a solution, but I think this is probably why our chakra-based header row was re-rendering despite no header changes.

If we wanted to optimize without using memo, we could split this context into 2  header row context and data row context. Or do some fancy hoisting like:
```
<Table>
  <TableHeaderRows columns={columns} />
  <TableContextProvider /*... */ />
    <TableDataRows />
  </TableContextProvider>
</Table>
```

Reply:
good point, however:
`id` would not change
`columns` occassional, however, i want a re-render in this case, since columns might be added or deleted
`rows` will not change since it's optimistic